feature/session/build.gradle

```diff
@@ -49,6 +48,8 @@ dependencies {
     implementation Dep.Picasso.picasso
     implementation Dep.Picasso.picassoTransformation
 
+    implementation Dep.Timber.android
```

Comment:
compile error is occur if it's removed. but i add dependency of Timber in ":ext:log" module. kotlin-native doesn't support Gradle dependency api?